Variant.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 23 11:56:17 2016
+
+@author: Mina Naghshnejad
+"""
+
+class Variant():
```

Comment:
I think at this point, we probably want to avoid defining any classes specific to the workflow config until we have a better idea of what we wrap up the workflow config proposal.